great_expectations_provider/operators/great_expectations.py

```diff
@@ -44,121 +78,404 @@ class GreatExpectationsOperator(BaseOperator):
+            self.batch_request = self.build_configured_sql_datasource_batch_request()
+        else:
             raise ValueError(
-                "Exactly one of checkpoint_name or checkpoint_config must be specified."
+                "Unrecognized, or lack of, runtime or conn_id datasource passed."
             )
 
-    def execute(self, context: Any) -> [CheckpointResult, dict]:
+    def build_default_action_list(self) -> List[Dict[str, Any]]:
+        """Builds a default action list for a default checkpoint."""
+        action_list = [
+            {
+                "name": "store_validation_result",
+                "action": {"class_name": "StoreValidationResultAction"},
+            },
+            {
+                "name": "store_evaluation_params",
+                "action": {"class_name": "StoreEvaluationParametersAction"},
+            },
+            {
+                "name": "update_data_docs",
+                "action": {"class_name": "UpdateDataDocsAction", "site_names": []},
+            },
+        ]
+
+        if (
+            os.getenv("AIRFLOW__LINEAGE__BACKEND")
+            == "openlineage.lineage_backend.OpenLineageBackend"
+            and self.use_open_lineage
+        ):
+            self.log.info(
+                "Found OpenLineage Connection, automatically connecting... "
+                "(This behavior may be turned off by setting use_open_lineage to False.)"
+            )
+            openlineage_host = os.getenv("OPENLINEAGE_URL")
+            openlineage_api_key = os.getenv("OPENLINEAGE_API_KEY")
+            openlineage_namespace = os.getenv("OPENLINEAGE_NAMESPACE")
+            if not (openlineage_host and openlineage_api_key and openlineage_namespace):
+                raise ValueError(
+                    "Could not find one of OpenLineage host, API Key, or Namespace environment variables."
+                    f"\nHost: {openlineage_host}\nAPI Key: *****\nNamespace: {openlineage_namespace}"
+                )
+            action_list.append(
+                {
+                    "name": "open_lineage",
+                    "action": {
+                        "class_name": "OpenLineageValidationAction",
+                        "module_name": "openlineage.common.provider.great_expectations",
+                        "openlineage_host": openlineage_host,
+                        "openlineage_apiKey": openlineage_api_key,
+                        "openlineage_namespace": openlineage_namespace,
+                        "job_name": f"validate_{self.task_id}",
+                    },
+                }
+            )
+        return action_list
+
+    def build_default_checkpoint_config(self):
+        """Builds a default checkpoint with default values."""
+        self.run_name = (
+            self.run_name
+            if self.run_name
+            else f"{self.task_id}_{datetime.now().strftime('%Y-%m-%d::%H:%M:%S')}"
+        )
```

Comment:
Could we just say `if not self.run_name: self.run_name = f"{self.task_id}_{datetime.now().strftime('%Y-%m-%d::%H:%M:%S')}"`?  Thanks.